llama-index-integrations/llms/llama-index-llms-you/llama_index/llms/you/base.py

```diff
@@ -0,0 +1,123 @@
+import os
+from typing import Any, Dict, Generator, Literal, Optional
+
+import requests
+import sseclient
+from llama_index.core.llms import (
+    CompletionResponse,
+    CompletionResponseGen,
+    CustomLLM,
+    LLMMetadata,
+)
+from llama_index.core.llms.callbacks import llm_completion_callback
+from pydantic import Field
```

Comment:
it's better to use our bridge here for pydantic:

`from llama_index.core.bridge.pydantic import Field`